tests/test_bundle.py

```diff
@@ -266,35 +269,64 @@ def _test_bundle_get_checkout(self, replica: Replica, test_fixtures_bucket: str)
                   .add_query("version", bundle_version)
                   .add_query("presignedurls", "true"))
 
+        @eventually(10, 2)
+        def test_creation_dates_updated():
```

Comment:
Is this a standalone or integration test? If I don't put a decorator on it, does it get run as standalone by default?

Reply:
Neither - it's a helper function defined inside `_test_bundle_get_checkout`

Reply:
Oh! sorry didn't notice that.

Usually I try to call such helpers `assert_blah` instead of `test_blah`.